www/lighthouse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+if(array_key_exists("HTTP_IF_MODIFIED_SINCE",$_SERVER) && strlen(trim($_SERVER['HTTP_IF_MODIFIED_SINCE']))) {
+  header("HTTP/1.0 304 Not Modified");
+} else {
+  include 'common.inc';
+  $ok = false;
+  if (isset($testPath) && is_dir($testPath)) {
+    $file = "${run}_lighthouse.html.gz";
+    $filePath = "$testPath/$file";
+    if (is_file($filePath)) {
+      $ok = true;
+
+      // Cache for a year
+      header('Last-Modified: ' . gmdate('r'));
+      header('Expires: '.gmdate('r', time() + 31536000));
```

Comment:
IIRC `Expires` is old and everyone supports `Cache-Control` now. Do we need it?

https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching?hl=en#cache-control